Tasks/Common/find-files-legacy/findfiles.legacy.ts

```diff
@@ -44,7 +44,7 @@ function extractPattern(patterns: string[], cwd: string): any {
     let excludePatterns: string[] = [];
 
     patterns.forEach(p => {
-        p.split(',').forEach(inputFilter => {
+        p.split(';').forEach(inputFilter => {
```

Comment:
Did you run UTs? You might need to fix tests?
Please add logic for escaping `;` by `;;` https://github.com/Microsoft/vsts-tasks/tree/master/Tasks/VsTest